java/arcs/core/host/EntityHandleManager.kt

```diff
@@ -95,7 +97,8 @@ class EntityHandleManager(val handleManager: HandleManager) {
         handleName: String,
         storageKey: StorageKey,
         schema: Schema,
-        handleMode: HandleMode = HandleMode.ReadWrite,
+        handleMode: HandleMode = HandleMode
+            .ReadWrite,
```

Comment:
LOL, sigh lint --fix did this crazy formatting

Reply:
It's... not ideal. And doesn't seem to happen on my machine. Can you revert the change?